### PR TITLE
Amoled theme

### PR DESCRIPTION
**Pull request by lance36**
theme optimized for amoled screens

### CODE REVIEW

**Comment by maartenpaauw** [comment]
Hello @lance36,

Thanks for your submission! Great work!
I'll leave some comments at the code.

Maarten

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 | Sublimination   | [MikaelSchultz](https://community.home-assistant.io/u/MikaelSchultz) posted [Sublimination Theme](https://community.home-assistant.io/t/sublimination-theme/67312)   | sublimination       |
 | Black and Green | [GreenTurtwig](https://community.home-assistant.io/u/GreenTurtwig) posted [Black and Green Theme](https://community.home-assistant.io/t/black-and-green-theme/28602) | teal                |
 | Vintage         | [surendran.anup](https://community.home-assistant.io/u/surendran.anup) posted [Vintage Theme](https://community.home-assistant.io/t/vintage-theme/42806)             | vintage             |
-| Amoled          | [lance36] posted [Amoled Theme](https://community.home-assistant.io/t/amoled-theme/94875)                                                                            | amoled             |
+| Amoled          | [lance36](https://community.home-assistant.io/u/lance36) posted [Amoled Theme](https://community.home-assistant.io/t/amoled-theme/94875)                                                                            | amoled             |
```
Comment (maartenpaauw) [inline README.md]: Could you please put your theme at the top of the list?
It's alphabetical ordered (based on variable name).

File: amoled.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+amoled:
+  # Main colors
```
Comment (maartenpaauw) [inline amoled.yaml]: Could you please remove all comments in this file?

File: amoled.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  primary-color: '#484A4C'                                                        # Header
```
Comment (maartenpaauw) [inline amoled.yaml]: Could you please replace all single quotes with double quotes in this file?

File: amoled.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  accent-color: '#E45E65'                                                         # Accent color
+  dark-primary-color: 'var(--accent-color)'                                       # Hyperlinks                                         
+  light-primary-color: 'var(--accent-color)'                                      # Horizontal line in about
+
```
Comment (maartenpaauw) [inline amoled.yaml]: Could you please remove all blank lines in this file?

File: amoled.yaml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  paper-slider-pin-color: 'var(--accent-color)'
+  paper-slider-active-color: 'var(--accent-color)'
+  paper-slider-container-color: 'linear-gradient(var(--primary-background-color), var(--secondary-background-color)) no-repeat'
+  paper-slider-secondary-color: 'var(--secondary-background-color)'
+  paper-slider-disabled-active-color: 'var(--disabled-text-color)'
+  paper-slider-disabled-secondary-color: 'var(--disabled-text-color)'
+
+  # Google colors
+  google-red-500: '#E45E65'
+  google-green-500: '#39E949'
```
Comment (maartenpaauw) [inline amoled.yaml]: Could you please add a newline at the end of this file?

**Review by maartenpaauw — CHANGES_REQUESTED** [review]

**Comment by lance36** [comment]
i think i fucked up a bit, trying to learn git :)

should be ok now, all changes suggested were made

**Comment by maartenpaauw** [comment]
> i think i fucked up a bit, trying to learn git :)
> 
> should be ok now, all changes suggested were made

@lance36 no problem! Learning is good!

There are now 2 newlines at the end of the file. You can remove 1. There is only 1 needed. 

When you removed the comments (at the end of the lines) you forgot to remove the leading spaces (check screenshot). 

screen shot 2019-01-28 at 14 15 54

If you fix these two things I can merge the PR.

Thanks again!

**Comment by lance36** [comment]
Done

File: amoled.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  label-badge-text-color: "var(--primary-text-color)"
+  label-badge-background-color: "#2E333A"
+  paper-button-ink-color: "#5294E2"
+  paper-button-color: "#5294E2"
+  paper-toggle-button-checked-button-color: "var(--accent-color)"
+  paper-toggle-button-checked-bar-color: "var(--accent-color)"
+  paper-toggle-button-checked-ink-color: "var(--accent-color)"
+  paper-toggle-button-unchecked-button-color: "var(--disabled-text-color)"
+  paper-toggle-button-unchecked-bar-color: "var(--disabled-text-color)"
+  paper-toggle-button-unchecked-ink-color: "var(--disabled-text-color)"
```
Comment (maartenpaauw) [inline amoled.yaml]: You missed one space :)

**Review by maartenpaauw — CHANGES_REQUESTED** [review]

**Comment by maartenpaauw** [comment]
Thanks for your time and submission 👍🏻

**Pull request merged**